modules/kernel/src/org/apache/axis2/Constants.java

```diff
@@ -325,6 +325,14 @@
 
     public static final String CUSTOM_SCHEMA_NAME_PREFIX ="customSchemaNamePrefix" ;
 
+    /**
+     * Constants used to resolve environment variables and system properties
+     */
+    public static final String SYS_PROPERTY_PLACEHOLDER_PREFIX = "$sys{";
+    public static final String ENV_VAR_PLACEHOLDER_PREFIX = "$env{";
+    public static final String PROPERTY_PLACEHOLDER_PREFIX = "${";
```

Comment:
May be DYNAMIC_PROPERTY_PLACEHOLDER_PREFIX would suite better, WDYT?

Reply:
Done in 51ff278a5422064f95aa588d469cd46c30f78ca7